src/Stripe.net/Entities/Source.cs

```diff
@@ -17,8 +18,9 @@ public class Source : StripeEntityWithId
         public SourceType Type { get; set; }
 
         public StripeAccount Account { get; set; }
-        public StripeDeleted Deleted { get; set; }
-        public StripeCard Card { get; set; }
         public StripeBankAccount BankAccount { get; set; }
+        public StripeCard Card { get; set; }
+        public StripeDeleted Deleted { get; set; }
+        public StripeSource SourceObject { get; set; }
```

Comment:
It's not possible to have a property with the same name as the wrapping class, so I named the property `SourceObject` instead of just `Source`.